src/commands/dev/dev.js

```diff
@@ -593,6 +611,7 @@ const createDevCommand = (program) => {
         .choices(['cache', 'mock', 'update'])
         .default('cache'),
     )
+    .addOption(new Option('--country <geoCountry>', 'Two letter country code').argParser(validateGeoCountryCode))
```

Comment:
I think it'd be good to throw if this flag is used without `--geo=mock`, but looking at [commander's documentation](https://github.com/tj/commander.js/) I couldn't find a good way to add a constraint like that.

Maybe @danez knows?

Reply:
If they pass the `--country` flag it is safe to assume they are mocking the geo data, since that is the only use case for that flag, so I think that is a valid approach.